src/client/search/categories/TeamCategory.js

```diff
@@ -0,0 +1,131 @@
+                    $(`.league.section[data-id="${league.id}"]`).removeClass('item filtered');
+                    const totalLeagues = $(`.league.item[data-id="${league.id}"]`).length;
+                    if (inputLength === 0) {
+                      $(`.league.count[data-id="${league.id}"]`).text(totalLeagues);
+                    } else {
+                      $(`.league.count[data-id="${league.id}"]`).text(`${leaguesShowing} / ${totalLeagues}`);
+                    }
+                  } else {
+                    $(`.league.section[data-id="${league.id}"]`).addClass('item filtered');
+                  }
+                }, 20);
+              });
+            }} />
+          </div>
+          <div className="ui styled accordion teams">
+            {leagueTeams.map(league => (
+              <div key={`leagueteams${league.id}`} data-id={league.id} className="league section">
+                <div className="title">
+                  <i className="dropdown icon"></i>
+                  {league.name.slice(0, league.name.length - 7)}
+                  <div className="ui purple horizontal basic label" style={{ float: 'right' }}>
+                    <span data-id={league.id} className="league count">{league.teams.length}</span>
+                  </div>
+                </div>
+                <div className="content">
+                  <div className="items">
+                    {league.teams.map(team => (
+                      <div key={`league${league.id}team${team.id}`}
+                        data-id={league.id} className="league item"
+                        style={{ cursor: 'pointer' }}
+                        onClick={() => this.props.onClickAddTeam(team.shortName)}
+                      >
+                        <div className="ui three column grid">
+                          <div className="column">
+                            <img className="ui avatar image" src={team.crestUrl} />
+                            {team.name}
+                          </div>
+
+                          <div className="center aligned column">
+                            <i className="add green circle icon float right"></i>
+                          </div>
+
+                          <div className="column">
+                            <div className="mini ui right floated purple button"
+                              onClick={(e) => {
+                                e.stopPropagation();
+                                this.props.onClickSelectTeam(
+                                team.id, team.name, team.shortName, team.crestUrl
+                                );
+                              }}
+                            >
+                              {team.name} Players
+                              <i className="right chevron icon"></i>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    ))}
```

Comment:
Again, this would be a lot more readable in its own \<LeagueTeam> component - would that mess up the jQuery in any way?